convert.py

```diff
@@ -79,6 +79,16 @@ def get_static_data(modelSpec):
         return {}
 
 
+def labelize(propName):
+    """
+    If a propName contains a `$`, we have to return a quoted label
+    """
+    if "$" in propName:
+        return "`" + propName + "`"
+    else:
+        return propName
```

Comment:
I suggest checking if `propName` matches a [simple-label](https://github.com/dhall-lang/dhall-lang/blob/1d2912067658fdbbc17696fc86f057d6f91712b9/standard/dhall.abnf#L125) regex `[a-zA-Z_][a-zA-Z0-9_/\-]*`. If not we quote